.github/workflows/check-build-test.yml

```diff
@@ -110,12 +111,12 @@ jobs:
           git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
           git checkout scratch
 
-      - name: Set up JDK ${{ matrix.jdk }}
-        # https://github.com/coursier/setup-action/releases
-        # v1.3.0
-        uses: coursier/setup-action@70323223454ac2a9eb2de46f389b4d045cbcdea5
+      - name: Set up JDK ${{ matrix.jdkVersion }}
+        uses: coursier/setup-action@v1.3.4
         with:
-          jvm: ${{ matrix.jdk }}
+          jvm: ${{ matrix.jvmName }}
+          # FIXME default index not giving us access to JDK 21 (yet)
```

Comment:
How this will be fixed? I see we are adding a FIXME to all builds. 
Will the cache be filled after the first run?

Reply:
Ah, wait. It's not about cache, but index.

From where comes that index and who maintains it?

Reply:
It's the source of the index normally used for coursier, just that it was not published yet. Note that I use a specific git hash so there should be no risk that it is replaced with something nasty.